src/custom/utils/analytics/themeEvents.ts

```diff
@@ -0,0 +1,9 @@
+import { Category, _reportEvent } from './index'
+
+export function toggleDarkModeAnalytics(darkMode: boolean) {
+  _reportEvent({
+    category: Category.THEME,
+    action: 'Toggle dark/light mode',
+    label: `${darkMode ? 'Light' : 'Dark'} mode`,
```

Comment:
isn't this weird? : if darkMode --> light

Maybe name the param something that makes it more clear how to use it. 
`activateLightMode`?